docs/plugins/cloud-gce.asciidoc

```diff
@@ -1,54 +1,53 @@
-Google Compute Engine Cloud Plugin for Elasticsearch
-====================================================
+[[cloud-gce]]
+=== GCE Cloud Plugin
 
-The GCE Cloud plugin allows to use GCE API for the unicast discovery mechanism.
+The Google Compute Engine Cloud plugin uses the GCE API for unicast discovery.
 
-In order to install the plugin, run: 
+[[cloud-gce-install]]
+[float]
+==== Installation
 
-```sh
-bin/plugin install elasticsearch/elasticsearch-cloud-gce/2.5.0
-```
+This plugin can be installed using the plugin manager:
 
-You need to install a version matching your Elasticsearch version:
+[source,sh]
+----------------------------------------------------------------
+bin/plugin install elasticsearch-cloud-gce
```

Comment:
+1 for thinking about people on 1.x for a while. They won't be able to follow these instructions (right?) and will need something else.